src/Microsoft.ML.Transforms/PermutationFeatureImportance.cs

```diff
@@ -0,0 +1,224 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using Float = System.Single;
+
+namespace Microsoft.ML.Transforms
+{
+    using Stopwatch = System.Diagnostics.Stopwatch;
+
+    internal sealed class PermutationFeatureImportance<TResult>
```

Comment:
Are there any plans to introduce the PFI command here?

Reply:
The priority is to get pfi in 0.8, which doesn't need the command. So pfi command would come later.

---
In reply to: [234781545](https://github.com/dotnet/machinelearning/pull/1658#discussion_r234781545) [](ancestors = 234781545)